apps/chrome-devtools/package.json

```diff
@@ -71,7 +71,7 @@
     "jest-preset-angular": "~14.2.0",
     "jsonc-eslint-parser": "~2.4.0",
     "minimist": "^1.2.6",
-    "nx": "~19.5.0",
+    "nx": "~19.8.0",
```

Comment:
why not moving to latest `20.1` version?

Reply:
there seem to be an issue with the nx-cloud cache on windows runner
we can try the major upgrade later on